src/test/java/org/assertj/core/api/Assertions_assertThat_with_Iterator_Test.java

```diff
@@ -67,7 +67,7 @@ public void should_create_Assert() {
   @Test
   public void should_initialise_actual() {
     Iterator<String> names = asList("Luke", "Leia").iterator();
-    Iterable<String> actual = (Iterable<String>) assertThat(names).actual;
+    Iterable<String> actual = (Iterable<String>) assertThat(names).toIterable().actual;
```

Comment:
this test class should only deal with `Iterator`, `actual` should be an `Iterator` and not an `Iterable`